app/views/questionnaire.py

```diff
@@ -173,12 +173,14 @@ def get_thank_you(eq_id, form_type, collection_id):  # pylint: disable=unused-ar
     logger.debug("theme selected", theme=theme)
     metadata = get_metadata(current_user)
     metadata_context = build_metadata_context(metadata)
+    page_title = 'We\'ve received your answers - {survey_title}'.format(survey_title=g.schema_json['title'])
```

Comment:
Welsh? I do wonder if these messages should be extracted so they can be updated easier if needed. Not sure